tools/find_untranslated_strings.py

```diff
@@ -0,0 +1,159 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from collections import namedtuple
+
+import re
+
+TransAndKey = namedtuple("TransAndKey", "translation, key")
+
+
+translation = re.compile(r'([a-z]{2}|zh-Han[st])\s*=\s*.*$', re.S | re.MULTILINE)
+
+ITUNES_LANGS = ['en', 'ru', 'ar', 'cs', 'da', 'nl', 'fi', 'fr', 'de', 'hu', 'id', 'it', 'ja', 'ko', 'nb', 'pl', 'pt', 'ro', 'sl', 'es', 'sv', 'th', 'tr', 'uk', 'vi', 'zh-Hans', 'zh-Hant']
+
+
+class StringsTxt:
+
+    def __init__(self):
+        self.translations = {} # dict<key, dict<lang, translation>>
+        self.comments_and_tags = {}
+        self.with_english = []
+        self.current_key = ""
+        self.all_langs = set()
+        self.duplicates = {} # dict<lang, TransAndKey>
+        self.keys_in_order = []
+        self._read_file()
+
+
+    def _read_file(self):
+        with open("../strings.txt") as strings:
+            for line in strings:
+                line = line.strip()
+                if not line: continue
+                if line.startswith("[["):
+                    self.keys_in_order.append(line)
+                    continue
+                if line.startswith("["):
+                    if line in self.translations:
+                        print("Duplicate key {}".format(line))
```

Comment:
Please, don't mix single quotes with double quotes in the same script. Use either single either double.